Calculator/Calculator/Operator.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Operator.swift
+//  Calculator
+//
+//  Created by 천승현 on 2023/01/27.
+//
+
+import Foundation
+
+enum Operator: Character, CaseIterable {
+    case add = "+"
+    case subtract = "-"
+    case divide = "/"
+    case multiply = "*"
+    
+    func calculate(lhs: Double, rhs: Double) -> Double {
+        switch self {
+        case .add:
+            return add(lhs: lhs, rhs: rhs)
+        case .subtract:
+            return subtract(lhs: lhs, rhs: rhs)
+        case .divide:
+            return divide(lhs: lhs, rhs: rhs)
+        case .multiply:
+            return multiply(lhs: lhs, rhs: rhs)
+        }
+    }
+    
+    private func add(lhs: Double, rhs: Double) -> Double {
+        return lhs+rhs
+    }
```

Comment:
```suggestion
    private func add(lhs: Double, rhs: Double) -> Double {
        return lhs + rhs
    }
```
연산자 양 옆으로는 띄어쓰기를 넣어주세요! 다른 연산도 마찬가지입니다.